schedulearn-frontend/src/components/NestedList/TopicAddModal.scss

```diff
@@ -0,0 +1,5 @@
+.ReactModal__Content {
```

Comment:
.topic-add-modal .ReactModal__Content selector, to apply specifically to your modal, not application wide.